Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -514,15 +514,52 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    /// SwifterSwift: Returns a localized string.
+    ///
+    ///        "Hello world".localized -> Hallo Welt
+    ///
+    var localized: String {
```

Comment:
Please remove this, it's unnecessary. Since you provide default parameter values callers can just run `.localized()` directly.

Reply:
If removing it, Swift will force users to add the `()`.